internal/services/network/bastion_host_resource_test.go

```diff
@@ -198,12 +199,14 @@ resource "azurerm_public_ip" "test" {
   resource_group_name = azurerm_resource_group.test.name
   allocation_method   = "Static"
   sku                 = "Standard"
+  zones               = ["1", "2", "3"]
 }
 
 resource "azurerm_bastion_host" "test" {
   name                = "acctestBastion%s"
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
+  zones               = azurerm_public_ip.test.zones
```

Comment:
The basic test config should only include properties that are `Required`, could you move this to the complete test config?

Reply:
Sure, will do that.
What should I do with the data source test? Initially included the field in the `Basic` because is the dataset used in that test, Should I update it to use the `Complete` instead?

Reply:
Yep I would update the data source test to use the complete config

Reply:
I made the changes (and updated the PR description accordingly)